src/module.ts

```diff
@@ -307,7 +306,7 @@ export default defineNuxtModule<ModuleOptions>({
   },
   async setup (options, nuxt) {
     const { resolve } = createResolver(import.meta.url)
-    const resolveRuntimeModule = (path: string) => resolveModule(path, { paths: resolve('./runtime') })
+    const resolveRuntimeModule = (path: string) => resolve('./runtime', path)
```

Comment:
Not that resolve just joint paths. It doesn't check extension. You might want to use resolver.resolePath  but have to try

Reply:
AFAIK almost in every cases of our usages Nuxt takes care of the extension matching and module works fine with `resolve` function

Reply:
If you want to rely on that (and it works with published module) surely!

Reply:
I've tested the build in a real project and it's fine. Will merge and check on edge channel too.
Thanks